app/assets/stylesheets/stores.css.scss

```diff
@@ -2,6 +2,16 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
+.create_store-wrapper h1{
+  font-size: 3em;
+  color: #E67E22;
```

Comment:
Color literals like `#E67E22` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E67E22` should be written as `#e67e22`